winml/test/common/googletest/main.cpp

```diff
@@ -0,0 +1,44 @@
+#include "testPch.h"
```

Comment:
How did you arrive at the new formatting, did you run `clang-format` on the file? Just curious if you used it or fixed it by hand.

Reply:
Yep! clang-format was ran in VS.